src/index.js

```diff
@@ -0,0 +1,76 @@
+import './style.css';
+
+const btnCity = document.getElementById('btnCity');
+const txtCity = document.getElementById('txtCity');
+const imgGif = document.getElementById('imgGif');
+const btnCelsius = document.getElementById('btnCelsius');
+const btnFahren = document.getElementById('btnFahren');
+const location = document.getElementById('location');
+const changeTemp = document.getElementById('changeTemp');
+const humidity = document.getElementById('humidity');
+const pressure = document.getElementById('pressure');
+const wind = document.getElementById('wind');
+const weather = document.getElementById('weather');
+
+const displayImage = () => {
+  fetch('https://api.giphy.com/v1/gifs/random?api_key=UATrdJQ14gXYSQb46ecz4AExyXbN27Qn&tag=&rating=G')
+    .then(response => response.json())
+    .then((response) => {
+      document.querySelector('img').src = response.data.images.original.url;
+    })
+    .catch((error) => {
+      document.querySelector('error').style.display = 'block';
+      document.querySelector('error').innerHTML = error;
+    });
+};
+
+imgGif.innerHTML = displayImage();
+
+let data;
+btnCity.onclick = () => {
+  const city = txtCity.value;
+  const KEY = '3200d53ac65b442eb5f439f5613ee06c';
+  const url = `https://api.openweathermap.org/data/2.5/weather?q=${city}&APPID=${KEY}`;
+  fetch(url).then(response => {
+    response.json().then(json => {
+      data = json;
+      getResponse(data);
+    });
+  });
+};
+
+function kToC(kTemp) {
```

Comment:
Oh, man! You did such a great job with the changes! But you forgot to implement this using ES6 syntax. 😢 
Please convert to arrow function syntax